keyboards/foxlab/leaf80/info.json

```diff
@@ -0,0 +1,111 @@
+{
+    "keyboard_name": "Leaf 80",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 18.25,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT_tkl_ansi": {
```

Comment:
Besides the ISO support the others mentioned, I think support for the WKL layouts (both ansi and iso) should be provided.